platformio.ini

```diff
@@ -19,6 +19,7 @@ default_envs = tbeam
 ;default_envs = rak4631_5005
 ;default_envs = rak4631_5005_eink
 ;default_envs = rak4631_19003
+default_envs = nano-g1
 ;default_envs = meshtastic-diy-v1
```

Comment:
Please don't edit the master platformio.ini unless absolutely necessary. This change can be reverted without harm.